app/screens/forgot_password/index.tsx

```diff
@@ -270,7 +256,7 @@ const ForgotPassword = ({componentId, serverUrl, theme}: Props) => {
         const unsubscribe = Navigation.events().registerComponentListener(listener, componentId);
 
         return () => unsubscribe.remove();
-    }, [dimensions]);
+    }, [componentId, dimensions, translateX]);
```

Comment:
```suggestion
    // eslint-disable-next-line react-hooks/exhaustive-deps treat translateX.value like useRef
    }, [componentId, dimensions.width]);
```

Reply:
Ref:

https://docs.swmansion.com/react-native-reanimated/docs/core/useSharedValue/#remarks

Reply:
Is not being modified, but ok